protos/guard/guard.proto

```diff
@@ -164,5 +165,31 @@ enum ResponseCode {
 message Result {
   ResponseCode code = 1;
   string message = 2;
-  google.protobuf.Timestamp response_time = 3;
+  google.protobuf.Timestamp response_time = 3 [(gogoproto.stdtime) = true];
+}
+
+message Command {
+  enum CommandType {
+    PREPARE_CHALLENGE = 0;
+    READY_CHALLENGE = 1;
+    DONE_CHALLENGE = 2;
+    START_DOWNLOAD = 3;
+    START_REPAIR = 4;
+    START_UPLOAD = 5;
+    ESTABLISH_CONTRACT = 6;
+    RESERVE1 = 7;
+    RESERVE2 = 8;
+    RESERVE3 = 9;
+  }
+  CommandType type = 1;
+  bytes s_val1 = 2;
+  bytes s_val2 = 3;
+  bytes s_val3 = 4;
+  int32 i_val1 = 5;
+  int32 i_val2 = 6;
+  int32 i_val3 = 7;
```

Comment:
more meaningful names would be great, or some comments on this? thanks

Reply:
same concern, not sure what these are

Reply:
I want to use this command object to store the command information which will be executed by other program, so I reserve this fields, s_val just mean string val while i_val is integer value, it rely on the command producer/consumer to understand which value was stored in which field.